Text/Touken Ranbu.txt

```diff
@@ -4774,12 +4774,247 @@ global.CardText[CardStat] = 'Touken Ranbu/Touken Danshi - Tsurugi/Over Trigger
 (You may only have one over trigger in a deck. When revealed as a trigger, remove that card, draw a card, choose one of your units, and it gets power +100 Million until end of turn! If revealed during drive check, activate its additional effect!)
 
 Additional Effect - Perform one of the following.
-� Choose one of your vanguards, and it gets critical +1/drive+1 until end of turn.
+� Choose one of your vanguards, and it gets critical +1 / drive +1 until end of turn.
 � Choose one of your rear-guards, and until end of turn, that unit performs drive check, and gets critical +1.'
```

Comment:
By the way, these lists are the only case where we do not put additional line breaks